docs/netstandard-20/packaging.md

```diff
@@ -0,0 +1,350 @@
+into a .NET Framework 4.6.1 project would end up adding hundreds of
+`<Reference>` elements to the project file, which is very noisy and thus a bad
+experience.
+
+To avoid this, we'll inject a single MSBuild `.targets` file that contains
+static references to all these assemblies.
+
+### Layout of .NET Standard (`NETStandard.Library`)
+
+In order to simplify packaging updates, we'll continue to make the
+`NETStandard.Library` package cumulative, which means it contains the assets for
+all previous .NET Standard versions. This makes it safe to always update this
+package to the latest version without losing support for platforms. However, we
+don't want to have to download the entire history of .NET Standard every time,
+which is why we'll split the package by TFM:
+
+    NETStandard.Library
+        ├───NETStandard.Library10
+        ├───...        
+        ├───NETStandard.Library16
+        └───NETStandard.Library20
+
+The idea is that we have a package whose ID is tied to a specific version of
+.NET Standard. Those packages are unlisted on NuGet and shouldn't be referenced
+by anyone, except the `NETStandard.Library` package which has a TFM specific
+dependency on those. For instance, the dependency on `NETStandard.Library16`
+is specific to the `netstandard16` TFM.
+
+So when running package restore for `netstandard16`, NuGet will only download
+two packages, `NETStandard.Library` as well as `NETStandard.Library16`. When the
+developer changes the TFM to `netstandard20` it will trigger another package
+restore which will then download `NETStandard.Library20` package instead.
+
+Each of the `NETStandard.LibraryXX` packages will have a structure like this:
+
+* `ref/netstandard{version}`
+    - Contains `netstandard.dll` reference assembly
+    - Contains `System.Runtime`-based facades, type forwarding to `netstandard.dll`
+    - Contains `mscorlib`-based facades, type forwarding to `netstandard.dll`
+* `ref/{platform}`
+    - Contains the placeholder `_._` so that no other platforms gets any compile
+      time assets as they are supposed to be provided by the platform.
+    - We will omit version in the TFM (effectively using 0) to avoid having to
+      rev this package every time the platform ships a new version
+
+For `NETStandard.Library20` we'll have to add the following additional assets
+in order to provide the support for .NET Framework 4.6.1:
+
+* `ref/net461`
+    - Contains the placeholder `_._` so that no assets are inserted by NuGet for
+      .NET Framework 4.6.1
+* `tools/net461`
+    - Contains a single `netstandard.targets` file that provides reference items
+      to all the facades
+    - Contains `netstandard` facade, type forwarding to `mscorlib.dll`
+    - Contains `System.Runtime`-based facades, type forwarding to `mscorlib.dll`     
+
+### Layout of contract packages (`System.*`)
+
+Our goal is to discontinue the existing fine-grained packages we have been using
```

Comment:
Fair, but I don't think these are specific to .NET Standard? It's mostly a function what happens for .NET Core.